packages/web/src/scss/themes/index.scss

```diff
@@ -0,0 +1,20 @@
+@use 'sass:map';
+@use '@themes' as themes;
+@use '../tools/themes' as themes-tools;
+
+$is-default-theme: true;
+
+@each $key, $value in themes.$themes {
+    $theme-colors: map.get($value, 'colors');
+    $theme-gradients: map.get($value, 'gradients');
+    $theme-shadows: map.get($value, 'shadows');
```

Comment:
Instead of hard-coding, you could put the property categories into a list and iterate over it:

```scss
$theme-categories: colors, gradients, shadows;

@each $category, $map in $theme-categories {
  // …
}
```

If plural vs. singular is a problem, we could strip the ending `s` from the category name here.